diff --git a/net/rfkill/rfkill-gpio.c b/net/rfkill/rfkill-gpio.c
index 76c01cb..4e32def 100644
--- a/net/rfkill/rfkill-gpio.c
+++ b/net/rfkill/rfkill-gpio.c
@@ -163,6 +163,7 @@ static int rfkill_gpio_remove(struct platform_device *pdev)
 static const struct acpi_device_id rfkill_acpi_match[] = {
 	{ "BCM4752", RFKILL_TYPE_GPS },
 	{ "LNV4752", RFKILL_TYPE_GPS },
+     { "OBDA8723",RFKILL_TYPE_BLUETOOTH },
 	{ },
 };
 MODULE_DEVICE_TABLE(acpi, rfkill_acpi_match);
---
diff --git a/drivers/staging/rtl8723bs/core/rtw_pwrctrl.c 
index 820a061506cc..80cf5a8b1557 100644
--- a/drivers/staging/rtl8723bs/core/rtw_pwrctrl.c
+++ b/drivers/staging/rtl8723bs/core/rtw_pwrctrl.c
@@ -34,7 +34,7 @@ void _ips_enter(struct adapter *padapter)
 
        if (rf_off == pwrpriv->change_rfpwrstate) {
                pwrpriv->bpower_saving = true;
-               DBG_871X_LEVEL(_drv_always_, "nolinked power save enter\n");
+               DBG_871X("nolinked power save enter\n");
 
                if (pwrpriv->ips_mode == IPS_LEVEL_2)
                        pwrpriv->bkeepfwalive = true;
@@ -73,7 +73,7 @@ int _ips_leave(struct adapter *padapter)
                if (result == _SUCCESS) {
                        pwrpriv->rf_pwrstate = rf_on;
                }
-               DBG_871X_LEVEL(_drv_always_, "nolinked power save leave\n");
+               DBG_871X("nolinked power save leave\n");
 
                DBG_871X("==> ips_leave.....LED(0x%08x)...\n", rtw_read32(padapter, 0x4c));
                pwrpriv->bips_processing = false;
---
